tests/test_fetch.py

```
@@ -0,0 +1,23 @@
+# tests/test_fetch.py
+
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)